src/components/LayerInfoDialog/LayerInfoDialog.vue

```diff
@@ -22,39 +22,53 @@
 
       <div class="px-2 py-2 flex-grow-1 overflow-y-auto">
         <v-card-text>
-          Lorem ipsum dolor sit amet, consectetur adipiscing elit.
-          Donec eleifend, mi id condimentum bibendum, purus enim iaculis lacus,
-          vitae posuere mauris risus non mauris. Vivamus eget mattis sem.
-          Nunc volutpat, nulla et semper elementum, magna odio sagittis ex,
-          vitae sodales ligula justo ut est. Donec semper metus odio,
-          id cursus neque cursus vel. Nunc sapien justo, convallis porta sollicitudin vitae,
-          porttitor nec ligula. Phasellus ullamcorper, nisl ac mattis gravida,
-          nibh sem sagittis quam, ut aliquet nulla est quis ligula.
-          Vestibulum molestie mi id libero commodo vehicula. Aliquam et porttitor ex.
-          Etiam posuere, nibh sit amet tristique mattis, lectus purus pretium dolor,
-          auctor dignissim tortor velit at urna. Suspendisse mi libero,
-          porta nec turpis ut, porta placerat eros. Proin ullamcorper,
-          eros ut vulputate egestas, nunc ex pharetra risus, et vulputate diam felis sed elit.
-          Integer eget egestas nisi, ac euismod erat.
+          <dl class="layer-info-dialog__metadata">
+            <div v-for="item in content" :key="item.key">
+              <dt class="font-weight-bold">
+                {{ item.key }}
+              </dt>
+              <dd v-html="item.value" />
+            </div>
+          </dl>
         </v-card-text>
       </div>
     </v-card>
   </v-dialog>
 </template>
 
 <script>
-  import { mapActions, mapGetters } from 'vuex'
-
   export default {
-    computed: {
-      ...mapGetters('data', [ 'layerDialogOpen' ]),
+    props: {
+      open: {
+        type: Boolean,
+        default: false,
+      },
+      title: {
+        type: String,
+        default: '',
+      },
+      content: {
+        type: Array,
+        default: () => [],
+      },
     },
-
     methods: {
-      ...mapActions('data', [ 'setLayerDialogOpen' ]),
-      onClickClose() {
-        this.setLayerDialogOpen({ open: false })
+      close() {
+        this.$emit('close')
       },
     },
   }
 </script>
+
+<style lang="scss">
+.layer-info-dialog__metadata {
+  display: grid;
+  grid-template-columns: auto 1fr;
+  row-gap: $spacing-smaller;
+  column-gap: $spacing-default;
+}
+
+.layer-info-dialog__metadata div {
```

Comment:
I would not style directly on elements, using a class is more future-proof.

Reply:
Only a `<div>` is allowed next to `<dt>`, and `<dd>` elements in an `<dl>`. There may not be other elements, so the `<div>` is not going to change. The `<div>` only serves as a tool for the `:key` prop. The content of this rule complements that (using `display: contents` makes it practically unusable for other styling. Adding a class can give the reader of the markup the idea that the div does more than 'just exist'

Reply:
Ok yes that makes sense 👍🏼